composer.json

```diff
@@ -11,18 +11,18 @@
         }
     ],
     "require": {
-        "composer/installers": "^1.2",
+        "composer/installers": "^1.9",
         "cweagans/composer-patches": "^1.6",
-        "drupal/core-composer-scaffold": "^8.8",
-        "drupal/core-project-message": "^8.8",
-        "drupal/core-recommended": "^8.8",
-        "localgovdrupal/localgov": "^1.0@alpha"
+        "drupal/core-composer-scaffold": "^8.9",
+        "drupal/core-project-message": "^8.9",
+        "drupal/core-recommended": "^8.9",
+        "localgovdrupal/localgov": "dev-feature/152-composer-2-fix"
```

Comment:
I am just wondering if you want to first make a new alpha release of localgovdrupal/localgov.  That should allow us to stick to **^1.0@alpha ** for localgovdrupal/localgov.

Reply:
Rest of the changes look fine to me.

Reply:
Yes - thanks @Adnan-cds  - that was a temporary change to allow testing these two together. I'll merge the other PR and tag a release, then switch this back... and who knows, tests might pass!